.github/workflows/pr_checks.yml

```diff
@@ -0,0 +1,27 @@
+name: "PR Checks"
+# Triggers the workflow on pull request events
+on: pull_request
+
+jobs:
+  pr_checks:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ 3.6, 3.7 ]
```

Comment:
do we still need python 3.6?

Reply:
We didn't move our jenkins agents yet to use higher python. So we will remove support for 3.6 once we will finish with that.  Discussed on last automation mtg.